scripts/geopmpy/io.py

```diff
@@ -1606,6 +1619,7 @@ def _try_float(val):
                 epoch_df = epoch_df.append(epoch_row, ignore_index=True)
 
                 app_row = copy.deepcopy(header)
+                app_row['FOM'] = figure_of_merit
```

Comment:
I'm wondering if this should even be added if figure_of_merit() returned None.  This would potentially add a lot of useless data if None was returned.

Reply:
yes, it adds an entire column of None

Reply:
I can remove this; any analysis script that uses the FOM will have to check if it's there (but the analysis won't work anyway if it's all None)